lnd/lnd_wallet.go

```diff
@@ -207,6 +208,22 @@ func (l *Client) CreateCoopSpendingTransaction(swapParams *swap.OpeningParams, c
 	return spendingTx.TxHash().String(), txHex, nil
 }
 
+// LabelTransaction labels a transaction with a given label.
+// This makes it easier to audit the transactions from faraday.
+// This is performed by LND's LabelTransaction RPC.
+func (l *Client) LabelTransaction(txID, label string) error {
+	txIDHash, err := chainhash.NewHashFromStr(txID)
+	if err != nil {
+		return err
+	}
+	_, err = l.walletClient.LabelTransaction(l.ctx,
+		&walletrpc.LabelTransactionRequest{
+			Txid:      txIDHash.CloneBytes(),
+			Label:     label,
+			Overwrite: true})
```

Comment:
I think we should include a reference (id) to the swap that the transaction relates to in the label.

Reply:
With reference to loopd, the following labels are set.

* `peerswap -- Opening(swap id=b171ee)`
* `peerswap -- ClaimByCoop(swap id=b171ee)`
* `peerswap -- ClaimByCsv(swap id=b171ee)`
* `peerswap -- ClaimByInvoice(swap id=b171ee)`